Libplanet.Tests/Net/SwarmTest.Preload.cs

```diff
@@ -830,297 +591,6 @@ async Task MineBlocks()
-                            currentCount++;
-
-                            if (currentCount == totalCount)
-                            {
-                                allProgressesReported.Set();
-                            }
-                        }
-                    }),
-                    trustedStateValidators: trustedPeers);
-
-                await allProgressesReported.WaitAsync();
-
-                Assert.Empty(receiverRenderer.ActionRecords);
-                Assert.Equal(minerChain.BlockHashes, receiverChain.BlockHashes);
-                int i = 0;
-                foreach (Address target in targets)
-                {
-                    foreach (BlockChain<DumbAction> chain in new[] { minerChain, receiverChain })
-                    {
-                        var chainType = ReferenceEquals(chain, minerChain) ? "M" : "R";
-                        IValue state = chain.GetState(
-                            target,
-                            stateCompleter: StateCompleters<DumbAction>.Reject
-                        );
-                        Assert.NotNull(state);
-                        Assert.Equal(
-                            $"({chainType}) Item0.{i},Item1.{i},Item2.{i},Item3.{i},Item9.{i}",
-                            $"({chainType}) {((Text)state).Value}"
-                        );
-                    }
-
-                    IValue TryToGetDeepStates() => receiverChain.GetState(
-                        target,
-                        deepBlockHash,
-                        StateCompleters<DumbAction>.Reject
-                    );
-
-                    Assert.Throws<IncompleteBlockStatesException>(
-                        () => TryToGetDeepStates()
-                    );
-
-                    i++;
-                }
-
-                foreach (BlockChain<DumbAction> chain in new[] { minerChain, receiverChain })
-                {
-                    IValue state = chain.GetState(
-                        genesisTarget,
-                        stateCompleter: StateCompleters<DumbAction>.Reject
-                    );
-                    Assert.NotNull(state);
-                    Assert.Equal((Text)"Genesis", state);
-                }
-
-                Assert.Equal(totalCount, downloadStates.Count);
-                i = 1;
-                foreach (StateDownloadState state in downloadStates)
-                {
-                    Assert.Equal(4, state.CurrentPhase);
-                    Assert.Equal(totalCount, state.TotalIterationCount);
-                    Assert.Equal(i++, state.ReceivedIterationCount);
-                }
-            }
-            finally
-            {
-                await StopAsync(minerSwarm);
-                await StopAsync(receiverSwarm);
-
-                fx1.Dispose();
-                fx2.Dispose();
-
-                minerSwarm.Dispose();
-                receiverSwarm.Dispose();
-            }
-        }
-
```

Comment:
I removed this test because it became unable to receive states from trusted peers.